games/b.yaml

```
@@ -583,6 +583,23 @@
   - 'https://raw.githubusercontent.com/cyxx/blues/master/blues1.png'
   - 'https://raw.githubusercontent.com/cyxx/blues/master/bbja2.png'
 
+- name: Blutfest
+  originals:
+  - Crimsonland
+  type: similar
```

```suggestion
  type: clone
```
according to project readme